build/deps/update-deps.py

```diff
@@ -423,17 +373,17 @@ def gen_crate(repo):
 
     prefix = f"{name}-{version['num']}"
     url = f"https://crates.io{version['dl_path']}"
-    extra_exports = f'\nVERSION = "{version["num"]}"'
 
-    return HTTP_ARCHIVE_TEMPLATE.format(
-        name=repo["name"],
-        url=url,
-        prefix=prefix,
-        sha256=version["checksum"],
-        macro_name=macro_name(repo),
-        extra_attrs=extra_attributes(repo),
-        extra_exports=extra_exports,
-        type="tgz",
+    return format_dep(
```

Comment:
Is a `crate` type dependency still useful for us to support given that we have `crates_universe`?